[WIP] Fix various documentation errors found across the website/docs directory, including typos, grammatical issues, and terminology inconsistencies.

- Do not need to update the cpsell config.
- Avoid making too many changes in a single PR; limit to less th... (#5946)
web-infra-dev/rsbuild: website/docs/en/community/releases/v1-0.mdx

For a long time, developers using webpack have been bothered by two major issues: **slow build times and configuration complexity**.

> We have used Rust to rewrite webpack into [Rspack](https://github.com/web-infra-dev/rspack), which addresses the slow build issue. However, to maintain compatibility with the webpack ecosystem, Rspack retains webpack's configuration and API, still resulting in some complexity and a learning curve.

Reviewer: ```suggestion
We have used Rust to rewrite webpack into [Rspack](https://github.com/web-infra-dev/rspack), which addresses the slow build issue. However, to maintain compatibility with the webpack ecosystem, Rspack retains webpack's configuration and API, which means it still has some complexity and a learning curve.
```
